### PR TITLE
Display fetched data and activate active button

### LINKED ISSUE #18: Fetch and display movies using the API

### LINKED ISSUE #19: Implement search to fetch and display searched movies

### PR DESCRIPTION
**Pull request by Mogle7Arkad**
This PR closes issues #18 and #19 . I crave your indulgence to review it. Best regards.

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -20,14 +20,14 @@ <h1>
...
             </form>
-            <ul class="nav-items">
-                <li class="nav-item">
-                    <a href="" class="nav-link">Home</a>
+            <ul class="nav-items" id="navItems">
+                <li class="nav-item active">
+                    <a href="" class="nav-link" id="home">Home</a>
                 </li>
-                <li class="nav-item">
+                <li class="nav-item" id="navItem">
```
Comment (tennjugu) [inline dist/index.html:27]: Ensure unique IDs for elements, 'navItem' is used twice on line 27 and line30

**Review by Bazzy09 — APPROVED** [review]

File: src/createhtmlAndApend.js
Comment (Bazzy09) [inline src/createhtmlAndApend.js:1]: This code is well-organized with modular components, clear class names, a fallback image for unavailable show images, consistent styling, however you can improve the code by enhancing flexibility by using function parameters for image source and show details.

**Review by Bazzy09 — APPROVED** [review]

File: src/tvshows.js
Comment (Bazzy09) [inline src/tvshows.js:1]: This code successfully fetches and displays a list of TV shows with good asynchronous handling and error reporting, good stuff sir.

**Review by tennjugu — APPROVED** [review]

**Comment by tennjugu** [comment]
Nice work 👍🏾

**Pull request merged**